EIPS/eip-4950.md

```diff
@@ -0,0 +1,78 @@
+---
+eip: 4950
+title: Entangled Tokens
+description: An ERC721-compatible standard with only two tokens minted that are tied together
+author: Victor Munoz (@victormunoz)
+discussions-to: https://ethereum-magicians.org/t/entangled-tokens/8702
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-03-28
+requires: 721
+---
+
+## Abstract
+We define entangled tokens to A and B to the tokens that share a same wallet, so that any transaction with the wallet could be operated by any of them.
+
+## Motivation
+The functionality is similar to the entanglement of two particles but with tokens, in the sense that whatever changes the state of A it affects B. Changes in the state of a token are actuated on a wallet by means of transfer to and from the wallet of whatever fungible and non-fungible tokens.
+
+Initially both minted tokens belong to the contract creator, but they can be transferred normally as any other NFT, so that they may belong to different users. The two users owning tokens can operate with the contract wallets. Therefore, the contract wallet (which may contain balances in several tokens) is “tangled” with both users. If a user makes some transfer, automatically the other user “sees” it.

```

Comment:
This is not appropriate content for the Motivation section. This section should explain why this EIP as a whole should exist. In this case, it should explain who needs entangled tokens, and why we need a standard interface for them.

Reply:
Updated